packages/editor/src/components/properties/PostProcessingSettingsEditor.tsx

```diff
@@ -275,11 +276,16 @@ export const PostProcessingSettingsEditor: EditorComponentType = (props) => {
     return value
   }
 
-  const setPropertyValue = (prop, val) => {
-    prop.set(val)
+  // trigger re-render - @todo find out why just setting the value doesn't trigger the reactor
+  // action: debounced the set property value
 
-    // trigger re-render - @todo find out why just setting the value doesnt trigger the reactor
+  const debouncedComposer = debounce(() => {
```

Comment:
debouncedComposer should be defined in module scope, then we can reduce this to about 200ms

Reply:
So define the debounce and use it from where the actual implementation is ? 
I'm on it